modules/ROOT/pages/entities-overview.adoc

```diff
@@ -0,0 +1,166 @@
+
+The search bar also offers quick filters, such as `Top 10 entities with highest error volume`, to show you common scenarios at a glance. You can also combine these with attributes and metrics to further specify the content that appears. 
+
+You can curate your search results for entity data using the following attributes: 
+
+* Environment 
+* Entity name 
+* Entity type 
+* Status
+* Response time 
+* Requests
+* Errors 
+* Client App (for APIs)
+* Locations (for APIs)
+* Org ID 
+
+You can curate your search results for entity data using the following metrics:
+
+tbd
+
+=== Applying Search Parameters
+
+include::partial$include-nav-steps-no-permissions.adoc[]
+
+include::partial$include-nav-entities.adoc[]
+
+. In the search bar, enter an attribute, element, business group, or environment you want to use to filter your data.
+
+=== Example of Applied Search Parameters
+
+tbd
+
+== Filtering Data using the Time Period Selector
+
+You can display the data points collected for the last given period of time (such as the last 5 or 30 minutes) or over a given date and time range. 
+
+Organizations with a Gold, Platinum, or Starter subscription can view up to 30 days of data. Organizations with a Titanium or Advanced subscription can view up to one year of data.
+
+=== Setting the Time Period to Display
+
+Use the drop-down in the calendar icon to select the time period to display in a graph.
+
+Your setting applies across all charts in the entity summary. 
+
+=== Changing the Time Period to Display
+
+You can display an earlier or later time period in your chart, and you can zoom in or out to cover a shorter or longer period of time.
+
+[cols="1,4"]
+|===
+| image:zoom-earlier-later.png[Zoom Out, Show Earlier and Later] a|
+To display earlier or later data, you can use the less-than (&lt;) and greater-than (&gt;) icons.
+
+To zoom out and display data over a longer time period, you use the magnifying glass icon located between the less-than and greater-than icons
+|===
+
+// steal Setting Data Display Options in a Dashboard and Setting the Time Period Display from dashboards-using.adoc but change the titles.
+
+// see if Changing the Time Period to Display still applies.
+
+// steal Refreshing Data
+
+
+== Understanding Summary and Performance Trends for Entities
+
+The *Entities Overview* page shows charts that detail entity performance. By default, the charts show data for entities in your current business group and environment. 
+
+=== Summary Charts
+
+Total Entities::
+The total number of entities according to your current filter conditions.
+
+Total Requests::
+The sum of inbound and outbound requests according to your current filter conditions.
+
+Average Latency::
+[I don't know] , calculated in milliseconds. 
+
+Average Error Rate::
+The percentage of requests that result in errors according to your current filter conditions.
+
+Active Alerts::
```

Comment:
we don't show Active Alerts for initial launch. We have Active Entities card instead.

Reply:
```suggestion
Active Entities::
```